### ISSUE TITLE
_config.yml,    powered:  enable: true   version: true

### ISSUE THREAD

**Issue by aixiu** [theme-next/hexo-theme-next#224]
- [ ] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [ ] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [ ] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
powered:
    # Hexo link (Powered by Hexo). 
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false

由 Hexo 强力驱动  is hide
### Actual behavior 
powered:
    # Hexo link (Powered by Hexo). 
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false

由 Hexo 强力驱动  Display normal

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
 node -v
v8.10.0

 npm -v
5.6.0


```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.6.0"
  },
  "dependencies": {
    "hexo": "^3.2.0",
    "hexo-deployer-git": "^0.3.1",
    "hexo-generator-archive": "^0.1.4",
    "hexo-generator-baidu-sitemap": "^0.1.2",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-feed": "^1.2.2",
    "hexo-generator-index": "^0.2.0",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-sitemap": "^1.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-leancloud-counter-security": "^1.2.0",
    "hexo-neat": "^1.0.4",
    "hexo-related-popular-posts": "^3.0.1",
    "hexo-renderer-ejs": "^0.3.0",
    "hexo-renderer-marked": "^0.3.0",
    "hexo-renderer-stylus": "^0.3.1",
    "hexo-server": "^0.2.0",
    "hexo-symbols-count-time": "^0.3.2"
  }
}


```

#### Hexo version

```
hexo: 3.6.0
hexo-cli: 1.1.0
os: Windows_NT 6.1.7601 win32 x64
http_parser: 2.7.0
node: 8.10.0
v8: 6.2.414.50
uv: 1.19.1
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2n
icu: 60.1
unicode: 10.0
cldr: 32.0
tz: 2017c

```

#### Hexo Configuration

```yml
  powered:
    # Hexo link (Powered by Hexo). 
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false

  theme:
    # Theme & scheme info link (Theme - NexT.scheme).
    enable: true
    # Version info of NexT after scheme info (vX.X.X).
    version: true
```

### NexT Information

**NexT Version:**

- [x] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [x] Gemini

#### NexT Configuration:

```yml

powered:
    # Hexo link (Powered by Hexo). 
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false
```

### Other Information 
enable: false or true The effect is the same.

$ git lg
*   24cdf82 - (HEAD -> master) Merge remote-tracking branch 'origin/master' (7 hours ago) <aixiu>
|\
| * 8fd7b81 - (origin/master, origin/HEAD) Update Crowdin configuration file (34 hours ago) <Ivan.Nginx>
| * adc76b7 - Renamed back `_en.yml` on `en.yml` and setted `default.yml` to link it. (34 hours ago) <Ivan.Nginx>
| *   c5ddac7 - Merge pull request #221 from theme-next/refactored-header-components (35 hours ago) <Ivan.Nginx>

**Comment by ivan-nginx** [comment]
1. No defined Hexo version.
2. No defined Hexo version.
3. I think need update to latest master branch for this feature.

**Comment by ivan-nginx** [comment]
Could you please provide the following information?

* Node.js and NPM version ($`node -v && npm -v`)?
* Hexo version ($`hexo -v`)?
* Your site package dependencies ($`cat package.json`)?

**Comment by ivan-nginx** [comment]
Default NexT config from latest master branch for now is:

```yml
footer:
  # Specify the date when the site was setup.
  # If not defined, current year will be used.
  #since: 2015

  # Icon between year and copyright info.
  icon:
    # Icon name in fontawesome, see: https://fontawesome.com/v4.7.0/icons
    # `heart` is recommended with animation in red (#ff0000).
    name: user
    # If you want to animate the icon, set it to true.
    animated: false
    # Change the color of icon, using Hex Code.
    color: "#808080"

  # If not defined, will be used `author` from Hexo main config.
  copyright:
  # -------------------------------------------------------------
  powered:
    # Hexo link (Powered by Hexo).
    enable: true
    # Version info of Hexo after Hexo link (vX.X.X).
    version: true

  theme:
    # Theme & scheme info link (Theme - NexT.scheme).
    enable: true
    # Version info of NexT after scheme info (vX.X.X).
    version: true
```

`hexo s`:

![image](https://user-images.githubusercontent.com/16944225/38606402-94ab1a78-3d7e-11e8-81a6-f7bb939c198b.png)

`hexo g & d` — same result as above, live demo [here](https://test.almostover.ru).

***

Can u show your online path of your NexT and/or Hexo configuration files?

**Comment by aixiu** [comment]
@ivan-nginx I just don't want to show Can not Reproduce，
```
powered:
    # Hexo link (Powered by Hexo). 
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false
```
`enable: false`  Always show,

**Comment by ivan-nginx** [comment]
Fixed in 8c139be commit.

**Comment by aixiu** [comment]
thx!
